### DISCUSSION TITLE
py_sql syntax suggestion

### DISCUSSION THREAD

**Discussion by tanshion** [rbatis/rbatis#427] [General]
类似以下这样的语法感觉更好阅读

```rust
#[py_sql2("
delete from biz_activity where delete_flag = 0
                  `if name != '':`
                     and name=#{name}
                  `if !ids.is_empty():`
                     and id in  ${ids.sql()}
")]
async fn py_exec(
    rb: &mut dyn Executor,
    name: &str,
    ids: &[i32],
) -> Result<ExecResult, Error> {
    impled!()
}
```

**Comment by zhuxiujia** [comment]
> 类似以下这样的语法感觉更好阅读
> 
> ```rust
> #[py_sql2("
> delete from biz_activity where delete_flag = 0
>                   `if name != '':`
>                      and name=#{name}
>                   `if !ids.is_empty():`
>                      and id in  ${ids.sql()}
> ")]
> async fn py_exec(
>     rb: &mut dyn Executor,
>     name: &str,
>     ids: &[i32],
> ) -> Result<ExecResult, Error> {
>     impled!()
> }
> ```

 ` 符号用于标记字符串，而不是标记语法
